masterlist.yaml

```diff
@@ -17752,12 +17752,17 @@ plugins:
         condition: 'many("The Skyrim conflict( Light| Medium| Hard)?( DE&O)?( Essential)?.*\.esp")'
         subs: [ 'The Skyrim conflict .esp file' ]
   - name: 'TheChoiceIsYours.esp'
+    url: [ 'https://www.nexusmods.com/skyrim/mods/26359' ]
     after: [ 'BetterQuestObjectives.esp' ]
     tag: [ Names ]
     clean:
       # version: 2.0
       - crc: 0x3A570202
         util: 'TES5Edit v4.0.1'
+    tag:
```

Comment:
There is already `tag: [ Names ]` a few lines above, so with your commit there are two `tag` elements for `TheChoiceIsYours.esp`. Replacing the existing `tag` with your new ones should do the trick.

Reply:
I had already wondered why LOOT only displays the `Names` tag in the UI... Interestingly enough, Wrye Bash saw all three tags.